javascript/app.js

```diff
@@ -99,11 +99,46 @@ FinancialFreedom.controller('ExpensesInputController', ['$scope', '$location', '
     };
 }]);
 
+FinancialFreedom.directive("btnOutputControl", function($filter){
+    return {
+        scope: {},
+        restrict: 'E',
+        templateUrl: 'partials/output_control.html'
+    }
+});
+
 FinancialFreedom.controller('TimeToRetirementController', ['$scope', 'RetirementCalculatorService', 'CreateRetirementGraphService', function($scope, RetirementCalculatorService, CreateRetirementGraphService) {
     
     var retirement_data = RetirementCalculatorService.calculateRetirementInfo();
+
+    $(function () {
+        $('[data-toggle="popover"]').popover();
+    });

```

Comment:
why is this wrapped in a jquery function and not simply call `$('[data-toggle="popover"]').popover();` outside?